src/features/unified-share-modal/flowTypes.js

```diff
@@ -393,6 +393,8 @@ export type USFProps = BaseUnifiedShareProps & {
     sharedLinkLoaded: boolean,
     /** Whether the FTUX tooltip should be rendered */
     shouldRenderFTUXTooltip: boolean,
+    /** Whether the new upgrade text should be rendered */
+    showNewUpgradeText: boolean,
```

Comment:
nit - it looks like this should be an optional type since the destructure in UnifiedShareForm is setting a default value. Also depending on how you're planning to populate this value, it might not be possible for it to be set when the USM is running as a standalone UI element and needs to make v2 API calls to do stuff